components/match_ticker/commons/match_ticker.lua

```diff
@@ -113,6 +116,13 @@ function MatchTicker:init(args)
 		enteredOpponentOnLeft = hasOpponent and Logic.readBool(args.enteredOpponentOnLeft or hasOpponent),
 		showInfoForEmptyResults = Logic.readBool(args.showInfoForEmptyResults),
 		onlyHighlightOnValue = args.onlyHighlightOnValue,
+		tiers = args.tiers and Array.filter(Array.parseCommaSeparatedString(args.tiers), function (tier)
+					local identifier = Tier.toIdentifier(tier)
+					return type(identifier) == 'number' and Tier.isValid(identifier)
+				end) or nil,
+		tierTypes = args.tiertypes and Array.filter(Array.parseCommaSeparatedString(args.tiertypes), function(tiertype)
+					return Tier.isValid(1, tiertype)
+				end) or nil,
```

Comment:
```suggestion
		tierTypes = args.tiertypes and Array.filter(Array.parseCommaSeparatedString(args.tiertypes), FnUtils.curry(Tier.isValid, 1)) or nil,
```

Reply:
fyi, that line will be too long for the linter ;)